ScoutSuite/providers/azure/configs/services.py

```diff
@@ -31,7 +31,7 @@ def __init__(self, metadata=None, thread_config=4, **kwargs):
 
         self.storageaccounts = StorageAccountsConfig(thread_config=thread_config)
         self.monitor = MonitorConfig(thread_config=thread_config)
-        self.sqldatabase = SQLDatabaseConfig(thread_config=thread_config)
+        self.sqldatabase = SQLDatabaseConfig()
```

Comment:
You'll have to make sure the user can still set how many resources the tool consumes.